go.mod

```diff
@@ -182,3 +182,5 @@ replace (
 
 // Exclude this version because the version has an invalid checksum.
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+replace github.com/elastic/elastic-agent-libs => github.com/AndersonQ/elastic-agent-libs v0.0.0-20231106171105-5bccf4058073
```

Comment:
[Blocker]
Merge https://github.com/elastic/elastic-agent-libs/pull/159 and release elastic-agent-libs then update the elastic-agent-libs version here